src/pages/about/index.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { Helmet } from "react-helmet";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import ImageHeader from "../../components/grid-aware/ImageHeader";
+import Spacer from "../../components/grid-aware/Spacer";
+import teamDesignWorkshopImage from "../../components/grid-aware/ThreeParagraphBlock/stories/image2.png";
```

Comment:
Let's not import images from the Storybook story. We can make a copy of the image for now, since this will all go away soon with the CMS integration.